md/multi-tenancy-and-tenant-configuration.md

```diff
@@ -56,34 +60,55 @@ After the first login, default theme will be loaded and the login page will look
 
 The Java PlatformAPI creates the tenant by updating the database and creating configuration based on the tenant template files (in database too). 
 The following example code uses the Engine Java APIs to create a tenant called "myNewTenantName":
-```groovy
-// Get platform login API using the PlatformAPIAccessor
-PlatformLoginAPI platformLoginAPI = PlatformAPIAccessor.getPlatformLoginAPI();
-// Log in to the platform
-PlatformSession platformSession = platformLoginAPI.login("platformAdmin", "platform");
-
-// Get the platform API
-PlatformAPI platformAPI = PlatformAPIAccessor.getPlatformAPI(platformSession);
-
-// Create a new tenant
-TenantCreator tenantCreator = new TenantCreator("myNewTenantName");
-tenantCreator.setUsername("install");
-tenantCreator.setPassword("install");
-long tenantId = platformAPI.createTenant(tenantCreator);
-platformAPI.activateTenant(tenantId);
-
-// Log out of the platform
-platformLoginAPI.logout(platformSession);
+```java
+    // Get platform login API using the PlatformAPIAccessor:
+    PlatformLoginAPI platformLoginAPI = PlatformAPIAccessor.getPlatformLoginAPI();
+    // Log in to the platform:
+    PlatformSession platformSession = platformLoginAPI.login("platformAdmin", "platform");
+    
+    // Get the platform API:
+    PlatformAPI platformAPI = PlatformAPIAccessor.getPlatformAPI(platformSession);
+    
+    // Create a new tenant:
+    TenantCreator tenantCreator = new TenantCreator("myNewTenantName");
+    tenantCreator.setUsername("install");
+    tenantCreator.setPassword("install");
+    long tenantId = platformAPI.createTenant(tenantCreator);
+    platformAPI.activateTenant(tenantId);
+    
+    // Log out of the platform:
+    platformLoginAPI.logout(platformSession);
 ```
 
 ### REST API
 
-The [platform REST API](platform-api.md) calls the Java PlatformAPI to create the tenant.
+The [platform REST API](platform-api.md) is a REST wrapper around the Java PlatformAPI to create the tenant.
 
 ## Tenant access
 
 A tenant is identified by an id, which is used to log in and to retrieve the tenant. A tenant also has a name. You can use the tenant name to retrieve the tenant id.
 
+To use the newly created tenant:
+```java
+    // use tenant-level api client:
+    APIClient apiClient = new APIClient();
+    
+    // login on the new tenant with the only existing 'technical' user, ...
+    apiClient.login(TENANT_ID, "install", "install");               // here, replace TENANT_ID by the ID of the tenant you just created above
```

Comment:
install/install doesn't sems to be the good default value since it's tenant #1 value